src/sdk/namespace_fs.js

```diff
@@ -1326,6 +1315,33 @@ class NamespaceFS {
         };
     }
 
+    _get_upload_info(stat) {
+        const etag = this._get_etag(stat);
+        const encryption = this._get_encryption_info(stat);
+        return {
+            etag,
+            encryption
+        };
+    }
+
+    _get_encryption_info(stat) {
```

Comment:
By the look of it, it will return only in gpfs case, if that is the case, can you add a comment on this function?

Reply:
is there a comment that will help you understand that better?

Reply:
It is pretty understandable, I think, but maybe add a comment:
`_get_encryption_info will return undefined if it is not in gpfs` 
or something like that 😄

Reply:
done